packages/hardhat-core/src/internal/constants.ts

```diff
@@ -18,6 +18,7 @@ export const HARDHAT_NETWORK_SUPPORTED_HARDFORKS = [
   "muirGlacier",
   "berlin",
   "london",
+  "arrowGlacier",
```

Comment:
We need to update the docs too.

Reply:
Addressed in https://github.com/nomiclabs/hardhat/pull/2124/commits/4ceff2dcffc9a254b5d8abde976f1a87b63b2d49